dongkyu/initial-settings-express/myapp/app.js

```diff
@@ -31,12 +30,65 @@ app.use(cors());
 app.use(morgan("dev"));
 
 
-//health check
 app.get("/ping", (req, res) => {
   return res.status(200).json({ message: "pong" });
 });
 
 
+app.post("/users", async (req, res) => {
+  const { name, age, email, password } = req.body
+
+  await appDataSource.query(
+    `INSERT INTO users(
+      name,
+      age,
+      email,
+      password
+    ) VALUES (?, ?, ?, ?);  
+    `,
+    [name, age, email, password]
+  );
+
+  res.status(201).json({ message : "userCreated" });
+})
+
+
+app.post("/posts", async (req, res) => {
+  const { title, content, user_id } = req.body
+
+  await appDataSource.query(
+    `INSERT INTO posts(
+      title,
+      content,
+      user_id
+    ) VALUES (?, ?, ?);
+    `,
+    [title, content, user_id]
+  );
+
+  res.status(201).json({ message : "postCreated"})
+})
+
+
+app.get("/users_posts", async (req, res) => {
```

Comment:
```suggestion
app.get("/posts", async (req, res) => {
```
어느 특정 유저의 게시글 목록을 조회하는 것이 아닌 db내 전체 게시글을 조회하는 것이므로 `/posts`를 `get`하는 것만으로 충분합니다! 또한 일반적으로 endpoint를 작성할 때에는 언더바(_)를 사용하지 않는 것이 범용적인 컨벤션입니다.